language/documentation/book/src/modules-and-scripts.md

```diff
@@ -86,13 +87,15 @@ module N { ... }
 
 Module names can start with letters `a` to `z` or letters `A` to `Z`. After the first character, module names can contain underscores `_`, letters `a` to `z`, letters `A` to `Z`, or digits `0` to `9`.
 
-```rust
+```move
 module my_module {}
 module FooBar42 {}
 ```
 
 Typically, module names start with an uppercase letter. A module named `MyModule` should be stored in a source file named `MyModule.move`.
 
 All elements inside a `module` block can appear in any order.
-Fundamentally, a module is a collection of [`types`](./structs-and-resources.md) and
-[`functions`](./functions.md). [Uses](./uses.md) import types from other modules. [Constants](./constants.md) define private constants that can be used in the functions of a module.
+Fundamentally, a module is a collection of [`types`](./structs-and-resources.md) and [`functions`](./functions.md).
+[Uses](./uses.md) import types from other modules.
+[Friends](./friends.md) specifies a list of trusted modules.
+[Constants](./constants.md) define private constants that can be used in the functions of a module.
```

Comment:
`define` -> `defines`

Reply:
Ah, maybe it should be the other way round, `specifies` -> `specify`

Reply:
Kinds of depending on whether we treat `[Uses](./uses.md)` as Uses (which is plural) or the document itself (which is singular)

Reply:
Oh now I see why I was confused. I think perhaps the linked words should not be plural. I misread them as the actual keyword names, which I was interpreting as singular.

Reply:
Addressed in the fixup! commit